openslides_backend/models/models.py

```diff
@@ -4,7 +4,7 @@
 from openslides_backend.models.base import Model
 from openslides_backend.shared.patterns import Collection
 
-MODELS_YML_CHECKSUM = "83dd0d1d74658e869a797bfebfd0fcc6"
+MODELS_YML_CHECKSUM = "fd53765503552b0562bd71ef2ff437d3"
 
 
 class Organization(Model):
```

Comment:
Please add `theme_ids`, since the themes need a link to the organisation or otherwise they are "lost" (not discoverable)

Reply:
Added `theme_ids`